index.js

```diff
@@ -49,5 +50,30 @@ module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, normalizeOptions(opts));
   },
-  version: version
+  version: version,
+
+  __debug: {
```

Comment:
Yeah, this is a good idea! It's clearly internal and we can change this later without worrying about breaking things.